### PR TITLE
feat(collector-create-page): apply keep-alive to steps

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
The values in the input forms from step 2 to step 4 will be retained. 
If you go back to step 1 and reselect the plugin, the form will be reset.

https://github.com/cloudforet-io/console/assets/50662170/df4e5036-1d34-4f09-a6af-a42bfb81d94e

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM! simple

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue [hunk 1]
```diff
@@ -142,6 +142,7 @@ watch([() => collectorFormState.provider, () => state.selectedRepository], async
 }, { immediate: true });
 
 onMounted(() => {
+    collectorFormStore.resetForm();
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue:145]: question: Is `onMounted` hook always executed?

**Review by WANZARGEN — APPROVED** [review]
LGTM!!!!! I love this PR

File: apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue:145]: Yes, It is!

**Pull request merged**